src/ImageSharp/Colors/Spaces/CieLab.cs

```diff
@@ -161,17 +161,17 @@ public override bool Equals(object obj)
         /// <inheritdoc/>
         public bool Equals(CieLab other)
         {
-            return this.AlmostEquals(other, Constants.Epsilon);
+            return this.backingVector.Equals(other.backingVector);
```

Comment:
Yeah happy to change this. As @antonfirsov noted correctly [yesterday](https://github.com/JimBobSquarePants/ImageSharp/pull/60#issuecomment-269468011) `Equals` and `AlmostEquals` should really be separate. My original code was buggy.